test/integration/admin/list/clinicians-all.js

```diff
@@ -35,7 +36,7 @@ context('clinicians list', function() {
 
         return fx;
       })
-      .visit('/clinicians')
+      .navigate('/clinicians')
```

Comment:
@luketlancaster for your mid-specify `visit` desires

Reply:
i knew if I made that mistake enough you'd figure out a way to make it so I wouldn't bug you with it anymore 🙃